kernel/src/main/scala/org/apache/toree/kernel/api/Kernel.scala

```diff
@@ -396,7 +372,29 @@ class Kernel (
     interpreterManager.interpreters.get(name)
   }
 
-  override def sparkSession: SparkSession = SparkSession.builder.getOrCreate
+  private lazy val defaultSparkConf: SparkConf = createSparkConf(
+    new SparkConf().setAppName(SparkKernelInfo.banner))
```

Comment:
@rdblue Could you please take a quick look at the comments above, particularly for the setAppName then we will be getting ready to merge.